ClangPowerTools/ClangPowerToolsShared/MVVM/Views/CompilerSettingsView.xaml

```diff
@@ -447,6 +447,32 @@
         Foreground="White"
         Style="{StaticResource SettingsButton}" />
       </WrapPanel>
+
+      <!--  Support for powershell 7  -->
+      <WrapPanel Grid.Row="1" Grid.Column="2">
+        <WrapPanel.ToolTip>
+          <ToolTip>
+            <TextBlock
+            MaxWidth="400"
+            Text="{Binding Tooltip.AliasPowerShellScripts}"
+            TextWrapping="Wrap" />
+          </ToolTip>
+        </WrapPanel.ToolTip>
+        
+        <TextBlock
+        Width="180"
+        Padding="10"
+        VerticalAlignment="Center"
+        FontFamily="SegoeUI"
+        FontSize="16"
+        Text="Powershell 7" />
+
+        <ToggleButton
+        HorizontalAlignment="Left"
+        VerticalAlignment="Center"
+        IsChecked="{Binding CompilerModel.Powershell7}"
```

Comment:
Please add a check for pwsh.exe availability when this toggle is enabled. In case pwsh.exe can't be found then inform the user to install PowerShell 7 or add to PATH and then retry.

Reply:
It's on my list already.
I just made some checks to try pwsh, to see if it's not changing the output a lot (it changes by the way, but this is another story)
Work is still in progress

Thank you for your feedback